docs/.vitepress/config/sidebar.ts

```diff
@@ -100,7 +100,8 @@ export const sidebar: DefaultTheme.Sidebar = {
         { text: 'Per-user Config', link:'/tips/config'},
         { text: 'Remote Cache', link: '/tips/cache'},
         { text: 'CI/CD Integration', link:'/tips/ci'},
-        { text: 'Multi-target Build', link:'/tips/multitarget'}
+        { text: 'Multi-target Build', link:'/tips/multitarget'},
+        { text: 'envd vs. other software', link:'/tips/comparison' }
```

Comment:
Please copy the file to zh and update https://github.com/tensorchord/envd-docs/pull/90/files#diff-28b8d9c36ba8331d833f98071811f63570f78ce49c485f289c73aa5417aa931dR44